### PR TITLE
feat(tools): provide instructions how to generate change files in batch

### PR DESCRIPTION
**Pull request by Hotell**
## Current Behavior

After migration is executed, user is missing instructions regarding change files. Based on lack of user knowledge of beachball apis, if we run migration on many packages we might end up manually going through beachball prompts which tend to be very tedious manual work. 

## New Behavior

After migration execution, if there are any changed files in particular package, generator will provide command that user should invoke on his convenience, which will create change files if needed for all packages that have been affected.

- This PR also adds more tests for tools and fixes redundant console spam created by userLogs

image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/275f5182f8008016c5bad0c00c38c52474a47b8b'>275f5182f8008016c5bad0c00c38c52474a47b8b</a></sub>
<div id="bundleSizeComment9423"/>

File: tools/generators/migrate-converged-pkg/schema.json [hunk 1]
```diff
@@ -27,5 +27,6 @@
       "pattern": "^@[/-\\w]+$"
     }
   },
-  "required": []
+  "required": [],
+  "additionalProperties": false
```
Comment (Hotell) [inline tools/generators/migrate-converged-pkg/schema.json:31]: this is true by default, we don't wanna allow random flags to be used

**Review by andrefcdias — APPROVED** [review]

File: tools/generators/generate-change-files.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+interface ChangeFileOptions
+  extends WithoutNullable<Pick<CliOptions, 'message' | 'type'>>,
+    Partial<NonNullable<Pick<CliOptions, 'dependentChangeType'>>> {}
+
+/**
+ *
+ * This is not intended to be used yet - added this here only for inspiration/reference.
+ *
+ * Beacbball determines changes only on staged files. To make this work as expected (good DX),
```
[the remark this answers is not in the document]
Comment (Hotell) [reply tools/generators/generate-change-files.ts:20]: bchhcbbcbc

[the remark this answers is not in the document]
Comment (Hotell) [reply tools/generators/generate-change-files.ts:20]: shame on me, I clicked merge without applying your suggestions :(

**Pull request merged**